CoreFoundation/Base.subproj/CFPriv.h

```diff
@@ -57,10 +57,12 @@ CF_EXPORT void _CFRuntimeSetCFMPresent(void *a);
 CF_EXPORT const char *_CFProcessPath(void);
 CF_EXPORT const char **_CFGetProcessPath(void);
 CF_EXPORT const char **_CFGetProgname(void);
+
+#if !TARGET_OS_WIN32
```

Comment:
We like to keep these in the positive sense if possible

Reply:
Hmm, would you prefer:

```
#if TARGET_OS_WIN32
#else
...
#endif
```

Trying to exhaustively enumerate the other targets seems like a failing exercise since most OSes are Unixy :-(.  But, if that really is preferable, I can attempt to do that.

Reply:
Hm, maybe it's ok. I'm not sure if we are super consistent about it anyway.